### DISCUSSION TITLE
Game of Life First Thoughts

### DISCUSSION THREAD

**Discussion by 777ParallelClematis** [Isaac-Smit/Conway-s-Game-of-Life-Group-1-#2] [General]
Any live cell with fewer than two live neighbors dies, as if by underpopulation.
Any live cell with two or three live neighbors lives on to the next generation.
Any live cell with more than three live neighbors dies, as if by overpopulation.
Any dead cell with exactly three live neighbors becomes a live cell, as if by reproduction

10 by 10 grid minimum

Need to access the 8 neighbouring grid

**Comment by 777ParallelClematis** [comment]
We are not allowed to talk on Teams. Teams is off limits. Punishable by forking.

**Comment by 777ParallelClematis** [comment]
![image_67141633](https://github.com/Isaac-Smit/Conway-s-Game-of-Life-Group-1-/assets/160546107/b337c0fc-355c-4756-9557-6464d24eed4e)

**Comment by 777ParallelClematis** [comment]
Study Room tomorrow at 12 - will book it tomorrow AM

**Reply by WilliamBus** [reply]
i may be able to come for a short while of time depending on when the wakefield bus leaves nelson

**Comment by WilliamBus** [comment]
I am not sure on where to put this message so ill put it in here, In the forfun.js I have made working user input code so the user is asked to put in their chosen number.

**Reply by WilliamBus** [reply]
now made working error checking aswell :))

**Reply by Isaac-Smit** [reply]
i tried to run yours and Erin's code, but im getting a 'MODULE_NOT_FOUND' error. not entirely sure what that means.

**Reply by 777ParallelClematis** [reply]
Thats an issue with your branches

**Comment by Isaac-Smit** [comment]
Have we thought about how we are gonna define a cycle of the game? will we make it fully dependent on user input regardless of if it is something or not?

**Reply by Isaac-Smit** [reply]
'it' being what ever the user enters.

**Reply by 777ParallelClematis** [reply]
I think make it dependent on user input. One input = one generation
